src-tauri/tests/fixtures/ftp/README-slowftp.md

```
@@ -0,0 +1,476 @@
+```python
+print("226 at %.1fs" % (time.time() - t0), rd())   # wrong
+```
+
+Python evaluates the `%` expression before calling `rd()`, so the timestamp was
+taken before the blocking read rather than after it. The reading was resolved by
+timestamping the **server** log, which showed the hold running its full 8.0s.
+When two ends disagree, the end you are not measuring is the one to believe.
+
+## The plumbing options
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)